README.md

```diff
@@ -27,6 +27,12 @@ The collection supports RHEL in-place upgrades for the following RHEL versions:
 
 The collection may be used for the RHEL upgrade paths and minor versions supported by the indicated upgrade utilities (Leapp or RUT). Refer the to Red Hat knowledge solution article [Supported in-place upgrade paths for Red Hat Enterprise Linux](https://access.redhat.com/articles/4263361) for the latest support details.
 
+Third-party products and packages are not upgraded. It is expected that the playbook including the `upgrade` role will perform the additional tasks required to handle the upgrade of any installed third-party tools and agents, for example [Veritas Cluster](https://www.veritas.com/support/en_US/doc/infoscale_wp_upgradewithRedHat), [SAP HANA](https://access.redhat.com/solutions/5154031), etc. Likewise for packages installed from non-RHEL repositories such as [Red Hat Software Collections](https://access.redhat.com/support/policy/updates/rhscl), [EPEL](https://docs.fedoraproject.org/en-US/epel/), [RPM Fusion](https://rpmfusion.org/), etc.
```

Comment:
I'm uncertain about mismatch of two sentences "Third-party products and packages are not upgraded" and then the opposite -  " playbook will perform <..> to handle the upgrade of any installed third-party tool and agents". There are two different things 1) upgrade of OS, and 2) upgrade of any third-party tools and agents, e.g SAP software/workloads. Those are different procedures and different tooling. This repo/playbook is for OS upgrade only, and particularly RHEL packages which shipped for SAP will be upgrade. So to avoid any confusion, I would keep it only as "It is expected that the playbook <...> will perform the upgrade of RHEL operating system. The installed applications and utilities, along with the configurations and preferences, are incorporated into the new version....<...>" 
On the contrary, EPEL or Fusion packages are not upgraded, but in that sentence, to me, it sounds like they do.

Reply:
is it possible to drop any wording about (how to) upgrade third party tooling? If a user upgrade an OS, it doesn't necessary mean they need/want to upgrade the third party tooling. E.g. in a SAP case, as I wrote earlier, upgrade of RHEL and upgrade of SAP installed there, are different procedures, different tooling, independent from each other. This repo for upgrade of OS only. I don't see how it can be tweaked to upgrade 3rd party (as SAP applications).

Reply:
> is it possible to drop any wording about (how to) upgrade third party tooling?...

I would rather not drop it, but I have reworded to make it less confusing and moved under "Not in scope" section so it's clear we are not talking about what the roles in this collection do.